package.json

```diff
@@ -35,10 +35,10 @@
   },
   "peerDependencies": {
     "cldr-data": ">=25",
-    "globalize": ">=1.1.0-rc.5"
+    "globalize": "^1.1.0"
   },
   "devDependencies": {
-    "babel": "^5.6.14",
+    "babel": "^6.3.26",
```

Comment:
Yeap, it would.

@scottgonzalez, @jzaefferer, do you have any suggestions for that please? jQuery UI [uses bower](https://github.com/jquery/jquery-ui/blob/master/Gruntfile.js#L256). Any projects doing this with npm that I could use as baseline?